Makefile

```
@@ -1,5 +1,31 @@
+
+.PHONY: test
+test:
+	go test -cover ./...
+
+.PHONY: coverage
+coverage:
+	go clean -testcache
 	go test -v ./... -covermode=count -coverpkg=./... -coverprofile coverage/coverage.out
 	go tool cover -html coverage/coverage.out -o coverage/coverage.html
+
+.PHONY: benchmark
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)